config/initializers/active_model_serializers.rb

```diff
@@ -0,0 +1,13 @@
+# frozen_string_literal: true
+
+# NOTE: Specify :json_api to comply with its spec.
+#   http://www.rubydoc.info/gems/active_model_serializers
+#   According to the official document ...
+#   By default ActiveModelSerializers will use the Attributes Adapter (no JSON root).
+#   But we strongly advise you to use JsonApi Adapter
+ActiveModelSerializers.config.adapter = :json_api
+# NOTE: AMS converts underscores to hyphens automatically to comply with recommendation of JSON API.
+#   See: http://jsonapi.org/recommendations/#naming
+#   The following setting disables that.
+#   It doesn't mean this is recommendation. fast_jsonapi doesn't do so, for instance.
+ActiveModel::Serializer.config.key_transform = :unaltered
```

Comment:
This is not required this time.
You can keep it though.

Reply:
@jp-ryuji ok (Y)

Reply:
@jp-ryuji 

If i remove it LOL

<img width="719" alt="screen shot 2018-04-03 at 19 43 27" src="https://user-images.githubusercontent.com/4110469/38249852-4fbab0e8-3777-11e8-85f9-fe5e01cde5c9.png">

Reply:
@thienhv 
That's a correct behavior.
You should change the expected response.